blocks/color-palette/index.js

```diff
@@ -77,12 +77,3 @@ export function ColorPalette( { colors, disableCustomColors = false, value, onCh
 		</div>
 	);
 }
-
-export default withContext( 'editor' )(
```

Comment:
This is a breaking change in how this component works, you need to provide the colors as a prop now. Should we provide a BC alternative?

Reply:
`ColorPalette` become a general purpose component after this refactoring. What we could do to provide backward compatibility is move `ColorPallete` to `components` module and still expose it in here with the context applied for the 2 upcoming release cycles. We would use `wp.components.ColorPallete` for the core blocks. What do you think?